bluemira/geometry/tools.py

```diff
@@ -1640,10 +1640,23 @@ def save_cad(
         shapes = list(shapes) if isinstance(shapes, Iterable) else [shapes]
     if names is not None and not isinstance(names, list):
         names = [names]
+    filename = Path(filename).as_posix()
 
+    if cad_format == "dagmc":
+        # for now
+        raise NotImplementedError(
+            "DAGMC export is not yet implemented in Bluemira. "
+            "Please use the FreeCAD GUI to export to DAGMC."
```

Comment:
Can this be done or should this really say
```suggestion
            "This will be enabled soon."
```